From: Michael Biebl <biebl@debian.org>
Date: Tue, 27 Nov 2018 18:36:27 +0100
Subject: [PATCH] Move D-Bus policy file to /usr/share/dbus-1/system.d/

To better support stateless systems with an empty /etc, the old location
in /etc/dbus-1/system.d/ should only be used for local admin changes.
Package provided D-Bus policy files are supposed to be installed in
/usr/share/dbus-1/system.d/.

This is supported since dbus 1.9.18.

https://lists.freedesktop.org/archives/dbus/2015-July/016746.html

https://gitlab.freedesktop.org/polkit/polkit/merge_requests/11

Imported from policykit-1_0.105-25.debian.tar.xz

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 data/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index f0beeba42cd2..ec41433b48d9 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -9,7 +9,7 @@ service_DATA     = $(service_in_files:.service.in=.service)
 $(service_DATA): $(service_in_files) Makefile
 	@sed -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
 
-dbusconfdir = $(sysconfdir)/dbus-1/system.d
+dbusconfdir = $(datadir)/dbus-1/system.d
 dbusconf_DATA = org.freedesktop.PolicyKit1.conf
 
 if POLKIT_AUTHFW_PAM
